ts/protobuf/index.ts

```diff
@@ -0,0 +1,3 @@
+import { signalservice as SignalService } from './SignalService';
+
+export { SignalService };
```

Comment:
This way we never have to do `import { signalservice as SignalService } …` while preserving source fidelity with `libsignal-service-java`: https://github.com/signalapp/libsignal-service-java/blob/f953ff5db7d4df732f8e3abf0d31b2859a51f595/protobuf/SignalService.proto#L7